ppr-ui/src/composables/address/BaseAddress.vue

```diff
@@ -227,7 +227,10 @@ export default defineComponent({
         addressLocal.value.region = ''
         addressLocal.value.postalCode = ''
       }
-      !props.triggerErrors && resetValidation()
+      // wait for schema update and validate the form
+      setTimeout(() => {
+        props.triggerErrors ? validate() : resetValidation()
+      }, 5)
```

Comment:
Thats pretty quick! 5milliseconds does the trick? 

If the intention is to wait for the call stack to clear, would an `await nextTick()` work prior to line 230?

Reply:
> Thats pretty quick! 5milliseconds does the trick?
> 
> If the intention is to wait for the call stack to clear, would an `await nextTick()` work prior to line 230 vs a timeout?

Reply:
Yea, even 5mls made a difference. I wanted to avoid making the whole function async, there was some tests failing, but let me try the nextTick().

Reply:
Just tried... not good, a bunch of validation tests are failing and waiting for nextTick() in them is not helping. If there is a desire to discuss this further and create a ticket to fix, let me know.

Reply:
No concern, thanks for trying!